scenarios/meta_conditions_module.cpp

```diff
@@ -50,13 +62,16 @@ class MetaConditions : public TickServer
 
             ret = BT_SUCCESS;
         }
-        else if( params == "BottleLocatedWithConfidenceX" )
```

Comment:
@barbalberto maybe we can keep 'BottleLocatedWithConfidenceX' to represent the "fake" one in Gazebo

Reply:
Should I add the `BottleLocatedWithConfidenceX` command to keep compatibility with something? with a configurable confidence parameter or without any parameter?

Reply:
From my side, as long as parameters are strings, they can be anything. They'll end up in a config file and read at SmartSoft module's startup, so feel free to use what you thing is more appropriate.